data/Sri Guru Granth Sahib Ji/0257.json

```diff
@@ -2050,7 +2050,7 @@
         "translations": {
           "English": {
             "Dr. Sant Singh Khalsa": {
-              "translation": "The Name of the Lord, shall not fall into hell.",
+              "translation": "the Name of the Lord, shall not fall into hell.",
```

Comment:
Recommended to reverse these translations as well. Current and suggested below.

Current:
```
ਨੰਨਾ; ਨਰਕਿ ਪਰਹਿ ਤੇ ਨਾਹੀ ॥
nµnw; nrik prih qy nwhI ]
NANNA: Those whose minds and bodies are filled with the Naam,

ਜਾ ਕੈ ਮਨਿ ਤਨਿ; ਨਾਮੁ ਬਸਾਹੀ ॥
jw kY min qin; nwmu bswhI ]
the Name of the Lord, shall not fall into hell.
```

Suggested
```
ਨੰਨਾ; ਨਰਕਿ ਪਰਹਿ ਤੇ ਨਾਹੀ ॥
nµnw; nrik prih qy nwhI ]
NANNA: Those shall not fall into hell,

ਜਾ ਕੈ ਮਨਿ ਤਨਿ; ਨਾਮੁ ਬਸਾਹੀ ॥
jw kY min qin; nwmu bswhI ]
whose minds and bodies are filled with the Naam, the Name of the Lord.
```